File: baekjoon_all/02000+/boj_2740.py
# Solve 2023-03-03
# Update 2023-10-09

import sys

input = lambda : sys.stdin.readline().rstrip()

n, m = map(int, input().split())
mat_a = [list(map(int, input().split())) for _ in range(n)]

m, k = map(int, input().split())
mat_b = [list(map(int, input().split())) for _ in range(m)]

mat_ans = [[0] * k for _ in range(n)]

for row in range(n):
    for col in range(k):
        for idx in range(m):
            mat_ans[row][col] += mat_a[row][idx] * mat_b[idx][col]

for row in range(n):
    print(*mat_ans[row])

File: baekjoon_all/02000+/boj_2740.cpp
// Solve 2022-08-03
// Update 2023-10-09

#include <bits/stdc++.h>
using namespace std;

#define FASTIO ios_base::sync_with_stdio(false);cin.tie(NULL); // boj_15552.cpp
#define SETPRECISION(n) cout << fixed;cout.precision(n); // boj_1008.cpp
#define SIZE(v) (int)v.size()
#define ALL(v) v.begin(),v.end()
using ll = long long;

int main() {
    FASTIO;

    int mat_a[100][100], mat_b[100][100], mat_ans[100][100];

    int n, m, k;
    cin >> n >> m;

    for (int row = 0; row < n; row++) {
        for (int col = 0; col < m; col++) {
            cin >> mat_a[row][col];
        }
    }

    cin >> m >> k;

    for (int row = 0; row < m; row++) {
        for (int col = 0; col < k; col++) {
            cin >> mat_b[row][col];
        }
    }

    for (int row = 0; row < n; row++) {
        for (int col = 0; col < k; col++) {
            for (int i = 0; i < m; i++) {
                mat_ans[row][col] += mat_a[row][i] * mat_b[i][col];
            }
        }
    }

    for (int row = 0; row < n; row++) {
        for (int col = 0; col < k; col++) {
            cout << mat_ans[row][col] << ' ';
        }

        cout << '\n';
    }

    return 0;
}
